sinonjs/samsam
# Use @sinonjs/eslint-plugin-no-prototype-methods

This PR replaces the local copy of the `no-prototype-methods` rule that relies on `eslint-plugin-local-rules` in favour of the version in `@sinonjs/eslint-plugin-no-prototype-methods` that can be shared among all the repositories.

See https://github.com/sinonjs/sinon/pull/2307


#### How to verify

1. Check out this branch
1. `npm install`
1. Change a file in `lib/` to use a prototype method, like `Array.prototype.sort(["one", "two"]);`
1. `npm run lint-js`
1. Observe that ESLint catches the error
